meta-balena-common/recipes-core/systemd/timeinit/timeinit-buildtime.service

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 Balena Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[Unit]
+Description=Set the system clock from the build timestamp
+DefaultDependencies=no
+ConditionPathExists=/etc/timestamp
+Wants=time-set.target
```

Comment:
Same conflict as in `fake-hwclock.service`

Reply:
See my comment regarding `fake-hwclock.service`